src/easyscience/Objects/Job/Analysis.py

```diff
@@ -1,18 +1,60 @@
 #  SPDX-FileCopyrightText: 2023 EasyScience contributors  <core@easyscience.software>
 #  SPDX-License-Identifier: BSD-3-Clause
 #  © 2021-2023 Contributors to the EasyScience project <https://github.com/easyScience/EasyScience
+from abc import ABCMeta
+from abc import abstractmethod
+from typing import Union
 
+import numpy as np
 
+from easyscience.Datasets.xarray import xr  # type: ignore
```

Comment:
Are we not moving away from xarray?

Reply:
We are, but conversion to scipp in libraries will be done gradually. I could remove the dependency on xarray in EasyScience, though, by deleting type hints.